attribute-projection-functions/src/main/java/org/hypertrace/core/attribute/service/projection/functions/Conditional.java

```diff
@@ -0,0 +1,13 @@
+package org.hypertrace.core.attribute.service.projection.functions;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public class Conditional {
+  @Nullable
+  public static String getValue(@Nonnull boolean condition,
+                                @Nullable String first,
+                                @Nullable String second) {
+    return condition ? first : second;
```

Comment:
I think we should probably always support nullable args. If condition is null, we can return null - but we never want to NPE.

Reply:
ok. got it. bad things happen to good people.